packages/hash/frontend/src/components/hooks/use-get-workspace-shortname-by-entity-uuid.ts

```diff
@@ -0,0 +1,75 @@
+import { extractBaseUri } from "@blockprotocol/type-system";
+import { systemUserShortname } from "@hashintel/hash-shared/environment";
+import { types } from "@hashintel/hash-shared/ontology-types";
+import { entityIdFromOwnedByIdAndEntityUuid } from "@hashintel/hash-subgraph";
+import { getRootsAsEntities } from "@hashintel/hash-subgraph/src/stdlib/element/entity";
+import { useEffect, useMemo, useState } from "react";
+import { useBlockProtocolGetEntity } from "./blockProtocolFunctions/knowledge/useBlockProtocolGetEntity";
+import { useGetWorkspaceByShortname } from "./use-get-workspace-by-shortname";
+
+export const useGetWorkspaceShortnameByEntityUuid = (params: {
```

Comment:
I’d slightly prefer `payload` instead of `params` here. `params` is plural, which suggests that it’d be an array.

```js
const [foo, bar] = params; // ? 🤔
const { foo, bar } = params; // ? 🤔

const { foo, bar } = payload; // 💡
```

I know TS helps us see the inners of the var, but this does not mean that we can't use additional cognitive hints to help our brains a little.

Reply:
If you feel strongly about this I think we should have an internal discussion, as we are using `params` in a lot of the TS code right now so I'd rather be consistent with this approach for now thank introducing a new concept such as `payload`.